apps/docs/content/guides/storage/s3/compatibility.mdx

```diff
@@ -0,0 +1,61 @@
+---
+id: 'storage-s3-compatibility'
+title: 'S3 Compatibility'
+description: 'Compatibility spec'
+subtitle: 'Learn about the compatibility of Supabase Storage with S3.'
+sidebar_label: 'S3'
+---
+
+Supabase Storage introduces S3 compatibility.
+This means you can use any S3 client to interact with your Storage product.
+
+<Admonition type="note">
```

Comment:
Do we have a link? A GitHub discussion thread would work.

Reply:
what link were you thinking?

Reply:
Anywhere people can leave their feedback! There are two `please let us know`s below. We shouldn't be encouraging people to just comment into a void, so whatever you'll be monitoring for feedback (Support queue, GitHub discussion, labelled GitHub issues), we'll need to link to it